core/src/androidMain/kotlin/Peripheral.kt

```diff
@@ -33,29 +36,42 @@ import kotlinx.coroutines.flow.asStateFlow
 import kotlinx.coroutines.flow.combine
 import kotlinx.coroutines.flow.first
 import kotlinx.coroutines.flow.onEach
+import kotlin.DeprecationLevel.WARNING
 import kotlin.coroutines.CoroutineContext
 
 private val clientCharacteristicConfigUuid = uuidFrom(CLIENT_CHARACTERISTIC_CONFIG_UUID)
-private val writeNotificationDescriptorDefault = Always
 
 public actual fun CoroutineScope.peripheral(
     advertisement: Advertisement,
 ): Peripheral = peripheral(advertisement.bluetoothDevice)
 
+public fun CoroutineScope.peripheral(
+    bluetoothDevice: BluetoothDevice,
+): Peripheral = AndroidPeripheral(coroutineContext, bluetoothDevice)
+
+@Deprecated(
```

Comment:
Are these already being used somewhere downstream that they need to be deprecated instead of just removed?

Reply:
They were available in `0.3.0` release, so just giving users of that version a notice/path to the new usage.